tests/phases/design/test_electrical_design.py

```diff
@@ -59,6 +61,78 @@ def test_parameter_sweep(distance_to_landfall, depth, plant_cap, cable):
     assert 1e6 <= o.total_substation_cost <= 1e9
 
 
+def test_calc_substructure_mass_and_cost():
+
+    o = ElectricalDesign(base)
```

Comment:
It'd be great if we could avoid single letter variables unless they're part of a formula.